backend/ordd_api/models.py

```diff
@@ -255,7 +255,7 @@ class Dataset(models.Model):
     is_existing = models.BooleanField()
     is_existing_txt = models.TextField(blank=True, null=False)
     is_digital_form = models.BooleanField()
-    is_avail_online = models.BooleanField()
+    is_avail_online = models.BooleanField("Is available Online")
```

Comment:
It was a test (and it works) a complete list of description will be provided in a different PR